config/components/manager-ocp/controller_manager_config.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: config.kueue.x-k8s.io/v1beta1
+kind: Configuration
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: :8080
```

Comment:
I think 0.11.0 is using :9443 now.

Reply:
I'm trying to match the settings mentioned for the Kueue operator. However, I don't see that change in the upstream main branch though: https://github.com/kubernetes-sigs/kueue/blob/main/config/default/metrics_service.yaml#L13

Reply:
https://github.com/openshift/kueue-operator/pull/31